Lib/multiprocessing/resource_tracker.py

```diff
@@ -159,7 +178,17 @@ def unregister(self, name, rtype):
         self._send('UNREGISTER', name, rtype)
 
     def _send(self, cmd, name, rtype):
-        self.ensure_running()
+        try:
+            self.ensure_running()
+        except ReentrantCallError:
+            # The code below might or might not work, depending on whether
+            # the resource tracker was already running and still alive.
+            # Better warn the user.
+            # (XXX is warnings.warn itself reentrant-safe? :-)
+            warnings.warn(
+                f"ResourceTracker called reentrantly for resource cleanup, "
+                f"which is unsupported. "
+                f"The {rtype} object {name!r} might leak.")
```

Comment:
@vstinner This might raise a warning from time to time during the test suite, will it fail the buildbots?

Reply:
Hum. If you run tests with -Werror, it will raises an UserWarning exception, and can make the tests fail, no?